README.md

```diff
@@ -328,7 +313,7 @@ No resources found.
 
 ## Upgrading
 
-In order to upgrade the CSI operator and driver from a previous version, perform [step 1](#1-download-the-manifest-from-github) and [step 4](#4-install-the-operator-while-using-a-user-defined-namespace) from [Installation](#Installation).
+To manually upgrade the CSI (Container Storage Interface) driver from a previous version with GitHub, perform [step 2](#1--install-the-operator) of the installation procedure for the latest version.
```

Comment:
I [don't think](https://github.com/IBM/ibm-block-csi-driver/blob/4d200fe1ccb6867cc4556cd206bda452bdcae65d/docs/csi_ug_upgrade.md) this is step 2 anymore

Reply:
yes yes, done